shared/actions/json/users.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
a new substore to hold information about users. We currently have this scattered all over the place but really it should all go here. TODO later move this info out of the tracker and into here more